.github/actions/setup-dfx/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Setup DFX'
+description: Setup DFX
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/cache@v2
+      with:
+        path: |
+          /usr/local/bin/dfx
+          ~/.cache/dfinity
+        key: dfx-cache-0.11.1-3
+    # This step hangs on Github actions on Darwin for some reason, that
+    # is why we run this only on Linux for now
```

Comment:
I'm being a little pedantic here but actions don't specify a platform to run on, so I think this comment is just confusing here. We could throw and error on Darwin; that being said, did you confirm that this still breaks on Darwin? I'd be happy to just remove the comment for now; we're not installing dfx on darwin anyway